recipe/build.sh

```diff
@@ -1,5 +1,5 @@
 mkdir build
 cd build
 export ISISROOT=$PWD
-cmake -GNinja -DJP2KFLAG=ON -Dpybindings=OFF -DKAKADU_INCLUDE_DIR=/isisData/kakadu -DbuildTests=OFF -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=$PREFIX ../isis
```

Comment:
We probably should leave the Kakadu include dir. I imagine it was from local testing?

Reply:
Hm I think there was a mix up during rebasing my 8.1 branch off of the official 8.1 branch but got this reverted, good catch.